src/Pdf/CakePdf.php

```diff
@@ -738,6 +752,40 @@ public function template($template = false, $layout = null)
     }
 
     /**
+     * Template path
+     *
+     * @param mixed $templatePath The path of the template to use
+     * @return mixed
+     */
+    public function templatePath($templatePath = false)
```

Comment:
Autsch, thats not good. We cannot change it due to BC, so your change is OK for now.
Cleanup has to be done in a future release.